tests/TestCase/Command/ModelCommandTest.php

```diff
@@ -602,9 +602,10 @@ public function testGetAssociationsConstraints()
                 'joinType' => 'INNER',
             ],
             [
-                'alias' => 'Users',
+                'alias' => 'Receivers',
```

Comment:
Would be nice if the aliases were always created based on the foreign key field, so `Senders` for `sender_id` too instead of `Users`.

Reply:
It seems to be that way - until table lookup.
The problem seems to be that

                    $found = $this->findTableReferencedBy($schema, $fieldName);
                    if ($found) {
                        $tmpModelName = Inflector::camelize($found);
                    }
                    
finds the users table, and as such then switches it back to Users